pubspec.yaml

```diff
@@ -22,4 +22,4 @@ dev_dependencies:
 
   build_runner: ^1.0.0
   build_test: ^0.10.0
-  build_web_compilers: '>=1.2.0 <3.0.0'
+  build_web_compilers: '>=1.2.0 <3.0.0'
```

Comment:
revert this change!

Reply:
I'm not accustomed to git but I tried to use Git Gui to amend the change. It's not visible [here](https://github.com/martinory/firebase-dart/commit/c0011bcda02931403dd02faf03c82b9e19d57c82) anymore.

Reply:
> I'm not accustomed to git but I tried to use Git Gui to amend the change. It's not visible [here](https://github.com/martinory/firebase-dart/commit/c0011bcda02931403dd02faf03c82b9e19d57c82) anymore.

But now there are a LOT of other files! Uh...